server/controllers/column.js

```diff
@@ -0,0 +1,53 @@
+const asyncHandler = require("express-async-handler");
+const Column = require("../models/Column");
+
+exports.getColumns = asyncHandler(async (req, res) => {
+  const columns = await Column.find();
+
+  return res.json(Columns);
```

Comment:
This will return a undefined error on the response. Make sure this is `columns` and not `Columns`

Reply:
Removing this function alltogether